content/docs/1.6.0/advanced-resources/rwx-workloads.md

```diff
@@ -31,22 +31,55 @@ It is necessary to meet the following requirements in order to use RWX volumes.
+
+For new volumes, users can override the defaults to use mount options of their choice by creating a customized StorageClass with an `nfsOptions` parameter. For instance:
+
+  ```yaml
+  kind: StorageClass
+  apiVersion: storage.k8s.io/v1
+  metadata:
+    name: longhorn-test
+  provisioner: driver.longhorn.io
+  allowVolumeExpansion: true
+  reclaimPolicy: Delete
+  volumeBindingMode: Immediate
+  parameters:
+    numberOfReplicas: "3"
+    staleReplicaTimeout: "2880"
+    fromBackup: ""
+    fsType: "ext4"
+    nfsOptions: "vers=4.1,noresvport,softerr,timeo=600,retrans=5"
+  ```
+
+  Replace the `nfsOptions` string with a customized comma-separated list of legal options.  Then, create RWX volume PVCs using the above StorageClass.
+
+### Notes
+
+1. The complete set of options must be supplied.
+
+2. If there is a typo or an erroneous value in the string, Longhorn does no validation of the override and will not catch it.  The mount will be rejected by the NFS server, and the volume cannot be created or attached.
+
+3. In versions 1.4.0 to 1.4.3 and 1.5.0 to 1.5.1 of Longhorn, the Longhorn CSI plugin uses a `hard` mount option within the share-manager Pod in the `NodeStageVolume` step. The `hard` mount allows NFS requests to persistently retry without termination, ensuring that IOs do not fail. When the server is back online or a replacement server is recreated, the IOs resume seamlessly, thus guaranteeing data integrity. However, there is potential risk that to maintain file system stability, the Linux kernel will not permit unmounting a file system until all pending IOs are written back to storage. The system cannot undergo shutdown until all such file systems are unmounted. If the NFS server fails to recover successfully, the client nodes must undergo a forced reboot.
+
+    To mitigate the issue for existing volumes in those releases, upgrade to v1.4.4, v1.5.2, or anything later.  The `persistentvolume.spec.csi.volumeAttributes` field is immutable, and making adjustments to the `nfsOptions` within this field is prohibited.  After upgrading the Longhorn system, the default `softerr` or `soft` option will be automatically applied when reattaching RWX volumes, if there were no overrides.
+
+4. The user may choose to mount `hard` using the nfsOptions override mechanism, for the advantages noted above.  If so, any such volumes will be subject to the related risks as well.
```

Comment:
```suggestion
4. The user may choose to mount `hard` using the nfsOptions override mechanism, for the advantages noted above. If so, any such volumes will be subject to the related risks as well.
```

I'm sure @jillian-maroket will catch most grammar related issues, but I saw this in a couple of places and don't think it generally fits with our documentation style.

Reply:
Two spaces is a habit from manual typewriter days (!).